dist/cfpb-chart-builder.css

```diff
@@ -0,0 +1,15 @@
+.cfpb-chart {
+  max-width: 650px;
+  height: 500px;
+}
+.cfpb-chart__tile-map {
```

Comment:
why not use the existing data-chart-type=tile_map attribute, instead of adding a new class here?